phantom/unit-test.js

```diff
@@ -1,5 +1,6 @@
+var system = require ('system');
```

Comment:
I think we can overlook this warning http://stackoverflow.com/questions/23603514/javascript-require-function-giving-referenceerror-require-is-not-defined